tests/models/test_fairchem.py

```diff
@@ -59,7 +59,7 @@ def test_fairchem_ocp_consistency(
 
     si_state = atoms_to_state([si_dc], device, torch.float32)
     # Get FairChem results
-    fairchem_results = fairchem_model(si_state)
+    fairchem_results = fairchem_model.forward(si_state)
```

Comment:
@orionarcher best add a comment here to explain that `.forward` was explicitly added for `codecov` to detect the `forward` being run and should not be removed